js/ng-views/edit.html

```diff
@@ -0,0 +1,8 @@
+<h3>Edit your insta</h3>
+<form>
+  <input value="vm.insta.author" data-ng-model="vm.insta.author"/>
```

Comment:
With `<input>`s, all you need is `data-ng-model` to populate the field -- no need for `value`.